docs/spec/staking/01_state.md

```diff
@@ -24,6 +24,19 @@ type Params struct {
 
 ## Validator
 
+Validators can have one of three statuses
+
+
+- `Unbonded`: The validator is not in the active set. They cannot sign blocks and do not earn 
+  rewards. They can receive delegations. Unbonding from an `Unbonded` validator is immediate.
```

Comment:
I'd to confirm that 
> Unbonding from an `Unbonded` validator is immediate.

is true within the code. This thinking has changed a couple of times